core/src/main/java/org/apache/accumulo/core/client/ZooKeeperInstance.java

```diff
@@ -117,7 +112,7 @@ public ZooKeeperInstance(String instanceName, String zooKeepers, int sessionTime
    */
   @Deprecated
   public ZooKeeperInstance(UUID instanceId, String zooKeepers) {
-    this(ClientConfiguration.loadDefault().withInstance(instanceId).withZkHosts(zooKeepers));
+    this(null, instanceId.toString(), zooKeepers, -1, new ZooCacheFactory());
```

Comment:
Some of these methods are deprecated. If we are at the point of altering their implementation, we should probably just remove them, rather than maintain them. In 2.0, we can drop anything that was deprecated in 1.9.0 or previous.

Reply:
I agree we could drop them but I don't really see the benefit yet.

Reply:
One benefit: no need to review this code change, because no need to change it. It's a bit hard to look at this change and see that it is the same behavior as before. I suspect it is different behavior, because the "loadDefault" reads files, and this does not look like it does. It would be better to drop the method than to change the behavior. Changing the behavior undermines the only reason why we've kept it: users not ready to change.

Reply:
Changing the implementation might break things but removing the methods will break anyone that uses them. I also don't think there is a major reduction in maintenance if they are removed. I could revert the changes and suppress deprecation warnings on the class.  Does that work for you?

Reply:
I reverted the behavior changes with latest commit.  I also minimized suppressions so they only apply to minimal amount of code.